scripts/assemble.sh

```diff
@@ -172,6 +172,19 @@ function remove_unneeded_files() {
     rm "$PATH_PLUGINS/opensearch-security/tools/install_demo_configuration.sh"
 }
 
+# ====
+# Get missing Wazuh tools and files into packages
+# ====
+function get_wazuh_files() {
+
+    local version=$(<VERSION)
+    version=${version%%.[[:digit:]]}
+
+    wget -q https://packages-dev.wazuh.com/4.9/config.yml -O $PATH_PLUGINS/opensearch-security/tools/config.yml
+    wget -q https://packages-dev.wazuh.com/4.9/wazuh-passwords-tool.sh -O $PATH_PLUGINS/opensearch-security/tools/wazuh-passwords-tool.sh
+    wget -q https://packages-dev.wazuh.com/4.9/wazuh-certs-tool.sh -O $PATH_PLUGINS/opensearch-security/tools/wazuh-certs-tool.sh
+}
+
```

Comment:
Several things here:

- Version is unused, as the files are always downloaded from the 4.9 branch.
- Line 180 raises a [ShellCheck warning](https://www.shellcheck.net/wiki/SC2155).
- Review function name and description. `add_wazuh_tools` would be preferable. Also, description would be better as: `Add additional tools into packages`.

Reply:
Fixed in 7ce89115fd5